src/src/pages/teams/index.js

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState, useMemo } from "react";
+import { MaterialReactTable } from "material-react-table";
+import {
+  Card,
+  CardTitle,
+  CardContent,
+  CardActions,
+  CardMedia,
+  Button,
+  Text,
+  Spinner,
+} from "@dfds-ui/react-components";
+import Page from "components/Page";
+import PageSection from "components/PageSection";
+import { useTeams } from "hooks/Teams";
+import SplashImage from "./splash.jpeg";
+import styles from "./teams.module.css";
+
+function Teams() {
+  const { teams, isLoading } = useTeams();
+
+  const columns = useMemo(
+    () => [
+      {
+        accessorKey: "name",
+        header: "Name",
+        size: 250,
+        Cell: ({ renderedCellValue }) => {
+          return (
+            <Text styledAs="action" as={"div"}>
+              {renderedCellValue}
+            </Text>
+          );
+        },
+      },
+      {
+        accessorKey: "description",
+        header: "Description",
+        size: 250,
+        Cell: ({ renderedCellValue }) => {
+          return (
+            <Text styledAs="action" as={"div"}>
+              {renderedCellValue}
+            </Text>
+          );
+        },
+      },
+      {
+        accessorKey: "capabilities",
+        header: "Capabilities Linked",
+        size: 250,
+        Cell: ({ renderedCellValue }) => {
+          return (
+            <Text styledAs="action" as={"div"}>
+              {renderedCellValue}
```

Comment:
I'm curious as to how this renders.
I am certain this is fine for now, but we will probably revisit this